components/Panel/HistoryPanel.tsx

```diff
@@ -0,0 +1,140 @@
+import { LoadingSkeleton } from "components";
+import { LoadingSpinner } from "components";
+import { VoteHistoryTable } from "components";
+import { black, green, red500 } from "constants/colors";
+import { formatNumberForDisplay } from "helpers";
+import { useUserContext, useVotesContext } from "hooks";
+import styled, { CSSProperties } from "styled-components";
+import { VoteT } from "types";
+import { PanelFooter } from "./PanelFooter";
+import { PanelTitle } from "./PanelTitle";
+import { PanelSectionText, PanelSectionTitle, PanelWrapper } from "./styles";
+
+export function HistoryPanel() {
+  const { getPastVotes, getIsFetching } = useVotesContext();
+  const { apr, cumulativeCalculatedSlash, cumulativeCalculatedSlashPercentage, userDataFetching } = useUserContext();
+  const bonusPenaltyHighlightColor = cumulativeCalculatedSlashPercentage?.eq(0)
+    ? black
+    : cumulativeCalculatedSlashPercentage?.gt(0)
+    ? green
+    : red500;
+
+  function isLoading() {
+    return getIsFetching() || userDataFetching;
+  }
+
+  return (
+    <PanelWrapper>
+      <PanelTitle title="History" />
```

Comment:
Style suggestion:
Make only the table scrollable so that the red and black header and the footer with the icons and "Powered by UMA" are always visible.
<img width="350" alt="image" src="https://user-images.githubusercontent.com/53186719/194029886-77007da7-65c6-494b-97d0-5a153d510e5e.png">